osquery/config/config.cpp

```diff
@@ -142,6 +158,11 @@ std::map<std::string, std::vector<std::string> >& Config::getWatchedFiles() {
   return getInstance().cfg_.eventFiles;
 }
 
+std::map<std::string, std::vector<std::string> >& Config::getYARAFiles() {
```

Comment:
nit, This should return a const ref since the shared lock will expire when it goes out of scope. The caller will still have a reference to the Config-internal data structure. ((Looking at some of the other accessor functions, they need const references too :crying_cat_face:.)) //cc @jedi22